test/gnuradio/test_adi_gnuradio.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import pyguit
+import time
+import os
+import shutil
+import xml.etree.ElementTree as ET
+from xml.dom import minidom
+
+
+
+class TestADIGnuradio:
+
+    @classmethod
+    def setup_class(self):
+        '''Setup the pyguit object. Will be called once before the start of test'''
+        self.gui = pyguit.gui()
+        self.gui.attach_openbox()
+        # self.gui.run_ssh_agent()
+        time.sleep(10)
+        try:
+            os.makedirs("results")
+            os.makedirs("logs")
+        except FileExistsError:
+            # directory already exists
+            shutil.rmtree('results')
+            shutil.rmtree('logs')
+            os.makedirs("results")
+            os.makedirs("logs")
+
+         # Initialize XML log file
+        self.gui.log_root = ET.Element("logs")
+        self.gui.log_tree = ET.ElementTree(self.gui.log_root)
```

Comment:
I did not see any reference to self.gui.log_tree, is this still necessary?

Reply:
I removed xml.dom since pytest already includes this feature.